src/gaussjacobi.jl

```diff
@@ -367,39 +364,216 @@ function feval_asy1(n::Integer, α::Float64, β::Float64, t::AbstractVector, idx
+    
+    # Higher terms (not needed for n > 1000).
+    tB2t    = tB2(t) 
+    A3t     = A3(t)
+    vals  .+= Jb.*tB2t/rho^5 + Ja.*A3t/rho^6
+    vals2 .+= Jbb.*tB2t/rho2^5 + Jab.*A3t/rho2^6
+
+    # Constant out the front (Computed accurately!)
+    ds = .5*(α^2)/n
+    s = ds 
+    jj = 1
+    while abs(ds/s) > eps(Float64)/10
+        jj = jj+1
+        ds = -(jj-1)/(jj+1)/n*(ds*α)
+        s = s + ds
+    end
+    p2 = exp(s)*sqrt((n+α)/n)*(n/rho)^α
+    g = [1, 1/12, 1/288, -139/51840, -571/2488320, 163879/209018880,
+         5246819/75246796800, -534703531/902961561600,
+         -4483131259/86684309913600, 432261921612371/514904800886784000]
+    f(z) = dot(g, [1;cumprod(ones(9)./z)])
+    C = p2*(f(n+α)/f(n))/sqrt(2)
+
+    # Scaling:
+    valstmp = C*vals
+    denom   = sin.(t/2).^(α+.5).*cos.(t/2).^(β+.5)
+    vals    = sqrt.(t).*valstmp./denom
+
+    # Relation for derivative:
+    C2     = C*n/(n+α)*(rho/rho2)^α
+    ders   = (n*(α-β .- (2n+α+β)*cos.(t)).*valstmp .+ 2*(n+α)*(n+β)*C2*vals2)/(2n+α+β)
+    ders .*= sqrt.(t)./(denom.*sin.(t))
+
+    return vals, ders
+end
+
+function asy2_higherterms(α::Float64, β::Float64, theta::AbstractVector)
+    # Higher-order terms for boundary asymptotic series.
+    # Compute the higher order terms in asy2 boundary formula. See [2]. 
+    
+    # These constants are more useful than α and β:
+    A = (0.25 - α^2)
+    B = (0.25 - β^2)
+    
+    # For now, just work on half of the domain:
+    c = max(maximum(theta), .5)
+    
+    # Scaled 2nd-kind Chebyshev points and barycentric weights:
+    t = .5*c*(sin.(pi*(-(nc-1):2:(nc-1))/(2*(nc-1))) .+ 1)
+    v = [.5; ones(nc-1)]
+    v[2:2:end] .= -1
+    v[end]     *= .5
+    
+    # The g's:
+    g   = A*(cot.(t/2) - 2 ./t) - B*tan.(t/2)
+    gp  = A*(2 ./ t.^2 - .5*csc.(t/2).^2) - .5*(.25-β^2)*sec.(t/2).^2
+    gpp = A*(-4 ./ t.^3 + .25*sin.(t).*csc.(t/2).^4) - 4*B*sin.(t/2).^4 .* csc.(t).^3
+    g[1]   = 0 
+    gp[1]  = -A/6-.5*B 
+    gpp[1] = 0
+    
+    # B0:
+    B0     = .25*g./t
+    B0p    = .25*(gp./t - g./t.^2)
+    B0[1]  = .25*(-A/6-.5*B)
+    B0p[1] = 0
+    
+    # A1:
+    A10   = α*(A+3*B)/24
+    A1    = .125*gp .- (1+2*α)/2*B0 .- g.^2/32 .- A10
+    A1p   = .125*gpp .- (1+2*α)/2*B0p .- gp.*g/16
+    A1p_t = A1p./t
+    A1p_t[1] = -A/720 - A^2/576 - A*B/96 - B^2/64 - B/48 + α*(A/720 + B/48)
+    
+    # Make f accurately: (Taylor series approx for small t)
+    fcos = B./(2*cos.(t/2)).^2
+    f = -A*(1/12 .+ t.^2/240+t.^4/6048 + t.^6/172800 + t.^8/5322240 +
+        691*t.^10/118879488000 + t.^12/5748019200 +
+        3617*t.^14/711374856192000 + 43867*t.^16/300534953951232000)
+    idx = t .> .5
+    f[idx] = A.*(1 ./ t[idx].^2 - 1 ./ (2*sin.(t[idx]/2)).^2)
+    f .-= fcos
+    
+    # Integrals for B1: (Note that N isn't large, so we don't need to be fancy).
+    C = (.5*c)*cumsummat
+    D = (2/c)*diffmat
+    I = (C*A1p_t)
+    J = (C*(f.*A1))
+    
+    # B1:
+    tB1    = -.5*A1p - (.5+α)*I + .5*J
```

Comment:
```suggestion
    tB1    = -A1p/2 - (1/2+α)*I + J/2
```
Is there any reason for using `.5*` instead of `/2`?
I thought `/2` is better for readability and type promotions.

Reply:
I saw that things like `α == 0.5` and `0.5 + α` are used elsewhere in the source, I imagine because they were translated from MATLAB. So I use `0.5`, `0.25`, etc to stay internally consistent. I don't feel strongly about it either way.

Reply:
Thank you for the explanation. I think it would be okay to keep something like `.5` for consistency in this PR.
Maybe I will update the constants in a future PR.